dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/RequestParams.java

```diff
@@ -48,6 +49,7 @@
  *
  * @author Giuseppe Nespolino
  */
+@OpenApi.Property
 @Data
 @NoArgsConstructor
 class RequestParams extends PagingAndSortingCriteriaAdapter
```

Comment:
I noticed that there are multiple classes with the same simple class name. We should make that unique.

Reply:
Why is that necessary?

These classes are package private so a good name can be reused without adding prefixes like we did before. The context of `enrollment`, `trackedentity`, ... is clear from the package so repeating that is redundant.